src/workloads/execution/CreateIndex.yml

```diff
@@ -0,0 +1,172 @@
+# This workload has 2 actors, InsertData and IndexCollection. InsertData inserts documents containing all types of indexes, and IndexCollection creates indexes for each of them, one at a time.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 150000
+    BatchSize: 10000
+    Document:
+      loc2d: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -180, max: 180}}]
+      loc2dSphere: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -90, max: 90}}]
+      point: { type: "Point", coordinates: [{^RandomInt: {min: -180, max: 180}}, {^RandomInt: {min: -90, max: 90}}] }
+      randomString: {^RandomString: {length: 50}}
+      randomInt: {^RandomInt: {min: -10000000, max: 10000000}}
+      "attributes": { randomString : randomString, randomString : randomString, randomString : { randomString :  randomInt, randomString :  randomInt }, randomString : { randomString : randomString }}
+  - &Nop {Nop: true}
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: IndexCollection
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 5
+    Database: *db
+    Operations:
+    - OperationMetricsName: Create2dIndexesCmd
+      OperationName: RunCommand
+      OperationCommand:
+        createIndexes: Collection0
+        indexes:
+        - key: 
+            loc2d: "2d"
+          min: -180
+          max: 180
+          bits: 32
+          name: 2dIndex
+    - OperationMetricsName: DropCreated2dIndexesCmd
```

Comment:
I wonder if it would be better to move all of this identical index building and dropping code in a separate YAML file and parameterized as Robert suggested?

Reply:
I'm not too sure about that since the build and drops are different for each index. I think Robert's suggestion was before I merged all the workloads and documents into one.